bodhi/server/bugs.py

```diff
@@ -201,15 +201,17 @@ def close(self, bug_id, versions, comment):
                 fixedin = [v.strip() for v in bug.fixed_in.split()]
                 # Strip out any empty strings (already stripped)
                 fixedin = [v for v in fixedin if v]
-                # And add our build if its not already there
-                if version not in fixedin:
-                    fixedin.append(version)
 
                 # There are Red Hat preferences to how this field should be
                 # structured.  We should use:
                 # - the full NVR as it appears in koji
                 # - space-separated if there's more than one.
-                args['fixedin'] = " ".join(fixedin)
+                fixedinString = " ".join(fixedin)
+                
```

Comment:
```flake8``` fails on this line because it contains whitespace:

```
rawhide	./bodhi/server/bugs.py:210:1: W293 blank line contains whitespace
rawhide	                
rawhide	^
```

If you are using the Vagrant development environment, you can reproduce this test by running ```btest```. You can also run ```flake8``` directly if you like.